app/views/feedbacks/new.html.erb

```diff
@@ -7,75 +7,81 @@
 <ul class="navigation actions" role="navigation">
-  <li><%= link_to ts("FAQ Index"), archive_faqs_path %></li>
-  <li><%= link_to ts("Tutorials"), admin_posts_path(:tag => 5) %></li>
-  <li><%= link_to ts("Release Notes"), admin_posts_path(:tag => 1) %></li>	
+  <li><%= link_to ts("FAQs & Tutorials"), archive_faqs_path %></li>
+  <li><%= link_to ts("Release Notes"), admin_posts_path(:tag => 1) %></li>
   <li><%= link_to ts("Known Issues"), known_issues_path %></li>
 </ul>
 <!--/subnav-->
 
 <!--main content-->
-<h3 class="heading"><%= ts("We love to hear from our users! Tell us about your experience - good, bad, or broken - using our site!") %></h3>
-<div class="userstuff">
-  <p><%= ts("We're still in our beta phase of development, so there are lots of bugs to be fixed and new features to be considered. For live updates on Archive performance, please check out our Twitter feed: <a href=\"http://twitter.com/ao3_status\">@AO3_status</a>. Please use this form to let us know about your experience while using the site - good and bad! This form is for questions about the Archive and the OTW. Questions for creators should be left on their works. If you have technical issues with this page, please use <a href=\"http://www.transformativeworks.org/contact_us\">the back-up form</a>.").html_safe %></p>
-  <p><%= ts("<strong>We can answer Support inquiries in %{list}.</strong> Please allow for up to one week delay for responses in any language other than English and español.", list: @support_languages.map(&:name).to_sentence).html_safe %></p>
-</div>
-<%= form_for(@feedback, html: { class: "post feedback" }) do |f| %>
-  <fieldset>
-    <legend><%= ts("Contact Information") %></legend>
-    <p class="notice" id="email-field-description">
-      <%= ts("Our spam filter does collect IP addresses, but we never see them.") %>
-    </p>
-    <dl>
-      <dt><%= f.label :username, ts("Your name (optional)") %></dt>
-      <dd>
-        <%= f.text_field :username %>
-      </dd>
-      <dt class="required"><%= f.label :email, ts("Your email (required)") %></dt>
-      <dd class="required">
-        <%= f.text_field :email, size: 60, "aria-describedby" => "email-field-description" %>
-      </dd>
-    </dl>
-  </fieldset>
+<% if @admin_setting.disable_support_form %>
+  <div class="userstuff">
+    <%= raw sanitize_field(@admin_setting, :disabled_support_form_text) %>
+  </div>
+<% else %>
+  <h3 class="heading"><%= ts("Please use this form for questions about how to use the Archive and to report any technical problems. We are always glad to hear from our users!") %></h3>
+  <div class="userstuff">
+    <p><%= ts("If your question or problem concerns potential violations of the Terms of Service, control of Orphaned Works, or issues requiring identity verification such as regaining access to a lost account, please <a href=\"abuse_reports/new\">contact our Policy and Abuse team</a> instead.").html_safe %></p>
```

Comment:
To minimize the HTML Translation will need to work with, we've been moving toward doing links with interpolation instead:

`<%= ts("If your question or problem concerns potential violations of the Terms of Service, control of Orphaned Works, or issues requiring identity verification such as regaining access to a lost account, please %{contact_abuse_link} instead.", contact_abuse_link: link_to(ts("contact our Policy and Abuse team"), new_abuse_report_path)).html_safe %>`

I was going to merge it anyway since it's not a huge deal, but I see it has merge conflicts still. Could you update this and the link in the next paragraph when you resolve those? Then this should be good to go.

Reply:
Thanks! I was in a rush to grab a small window of opportunity so went with the simple link, but this is much better. Updated and also changed the date on the migration.

Reply:
Oh, yay, thanks for updating the migration as well! It looks like the Twitter link in the next paragraph didn't get updated to the new format, but since LO is trying to wrap up testing pre-hurricane, I'm going to merge this anyway.